internal/commands/user.go

```diff
@@ -60,7 +59,6 @@ func getUsersIDsInConversation(
 			app.Logger.Error(
 				ctx,
 				"command/user.getUsersIDsInConversation",
-				log.NewValue("channelID", channelID),
```

Comment:
Why removing the ChannelID from this log but log adding a logWriter with it?

Reply:
Also, add a log.Action("GetUsersInConversationContext") to identify in what part of the func the error was thrown.